docs/docs/server-rendering.md

```diff
@@ -49,6 +49,46 @@ export default () => ({
 });
 ```
 
+`getHeadTags` can be called multiple times. Each time it will only return the new tags since last called. This can be helpful when head tags are added dynamically when rendering.
+
+For example, you may want to send back an initial response before you are done rendering your response:
+
+```js
+import { initialResponseTemplate, followupResponseTemplate } from './template';
+import middleware from './middleware';
+
+export default () => ({
+  renderCallback: ({ SkuProvider, getBodyTags, getHeadTags }, req, res) => {
+    res.status(200);
+    // Call `getHeadTags` early to retrieve whatever tags are available.
+    res.write(initialResponseTemplate({ headTags: getHeadTags() }));
+    res.flush();
+    await Promise.resolve();
```

Comment:
I think this is confusing. Would rather just have a comment saying something like `Perform required data fetching` or similar.

Reply:
It's actually a required behaviour to allow Node to release the event loop and actually perform the flush that it said it has flushed. You'll need to do something like this for any flush call.
The other option is to use
```js
await res.flush()
```

but that's a bit deceitful, as it wouldn't be obvious that res.flush doesn't return a promise and your just saying 'seriously Node, why are you so single minded'